pep-0008.txt

```diff
@@ -1409,18 +1464,18 @@ similar to those on function annotations described above:
 - There should be no space before the colon.
 
 - If an assignment has a right hand side, then the equality sign should have
-  exactly one space on both sides.
-
-- Yes::
+  exactly one space on both sides::
 
+      # Correct:
```

Comment:
Again, let's try a blank line after the Correct/Wrong comment here.

Reply:
I'll add it.